rsconnect/bundle.py

```diff
@@ -2015,7 +2015,7 @@ def describe_manifest(
 
 
 def write_quarto_manifest_json(
-    directory: str,
```

Comment:
I know naming probably isn't in scope for feedback, but `path` covers `file_or_directory` nicely. Feel free to disregard. (I know we already used `file_or_directory` before, and perhaps widely in `rsconnect-python`, so if this is a settled issue, definitely disregard ☺️)

Reply:
This is an existing name and used as the argument name for `rsconnect write-manifest quarto` and `rsconnect deploy quarto`. We used that name to make it clearer that the target  could be either a directory or a single file. This was different from the other commands at the time.